org.osgi.test.junit5/pom.xml

```diff
@@ -99,6 +99,10 @@
 			<groupId>org.osgi</groupId>
 			<artifactId>org.osgi.service.log</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>org.osgi.service.cm</artifactId>
```

Comment:
I don't think we want to make the test artifacts from this projects dependent on many different OSGi API packages. This PR makes org.osgi.test.junit5  dependent on Config Admin at runtime (it will need to import the package). This makes org.osgi.test.junit5 awkward to use when I am not even testing or using CM. I think we will need to make individual artifacts for each new OSGi API that we want to test. Similar to how we have different artifacts for the assertj tests.